Formula/now-cli.rb

```diff
@@ -0,0 +1,17 @@
+class NowCli < Formula
+  desc "The command-line interface for realtime global deployments."
+  homepage "https://zeit.co/now"
+  url "https://github.com/zeit/now-cli/releases/download/7.0.2/now-macos"
```

Comment:
I thought [`bottle :unneeded`](http://www.rubydoc.info/github/Homebrew/brew/Formula.bottle) is for prebuilt package, did I get it wrong?

And the now-cli was originally on npm registry, which mainly download a prebuilt binary while installation, the team at Zeit also create [zeit/pkg](https://github.com/zeit/pkg) for this, for "No need to download hundreds of files via npm install to deploy your application. Deploy it as a single file"(one of [Use Cases](https://github.com/zeit/pkg#use-cases)), I think it was meant to be installed as a prebuilt binary.

ping @leo Just in case I have any misunderstanding :D